AMP-87690 Add IP range for CDP related help doc (#47)
amplitude/amplitude-dev-center: docs/data/destination-event-streaming-overview.md

2. **RefaceAI:** Utilized Amplitude's Kinesis Data Stream integration for real-time event piping from mobile clients to back-end services. This avoided the complexity of building and maintaining a custom pipeline.
3. **Invoice Simple:** Leveraged Event Streaming for a robust engagement campaign, customizing messaging based on a series of events to enhance engagement effectiveness.

### What's the IP range of your service?

> The IP addresses based on whether you're using EU or US Amplitude data centers.

Reviewer: ```suggestion
Amplitude data centers use the following IP addresses, depending on their region:
```

> - Amplitude US IP address

Reviewer: ```suggestion
- Amplitude US IP addresses
```

    - 52.33.3.219
    - 35.162.216.242
    - 52.27.10.221

> - Amplitude EU IP address

Reviewer: ```suggestion
- Amplitude EU IP addresses
```
